### PR TITLE
drop failing e2e quick app

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

this test is failing for long time and even the code lives outside this repo so not in direct control

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @carmocca @borda

### CODE REVIEW

File: .github/CODEOWNERS
Comment (Borda) [inline .github/CODEOWNERS:50]: alphabetic order

File: .github/CODEOWNERS
Comment (Borda) [inline .github/CODEOWNERS:14]: let's authors decide what shall be here

**Comment by awaelchli** [comment]
I ran the quick start app manually when submitting this PR a few weeks ago: https://github.com/Lightning-Universe/lightning-quick-start/pull/30 and it ran fine. Do you think the issue is here in our testing?

File: .github/CODEOWNERS
Comment (awaelchli) [reply .github/CODEOWNERS:50]: I don't think I should be first. Thomas knows more than me in that code base. The ordering should probably reflect that

**Review by awaelchli — APPROVED** [review]
I am not sure whether we should really skip this app. It seems to be working fine when run manually as a regular app.

**Review by ethanwharris — APPROVED** [review]

**Review by carmocca — APPROVED** [review]
Codeowner unblock

File: .github/CODEOWNERS
Comment (Borda) [reply .github/CODEOWNERS:50]: alphabetic

File: .github/CODEOWNERS
Comment (Borda) [inline .github/CODEOWNERS:54]:
```suggestion
/src/lightning/app                          @tchaton @lantiga @awaelchli @ethanwharris
/src/lightning_app                          @tchaton @lantiga @awaelchli @ethanwharris
/tests/tests_app                            @tchaton @lantiga @awaelchli @ethanwharris
/tests/integrations_app                     @tchaton @lantiga @awaelchli @ethanwharris
/examples/app_*                             @tchaton @lantiga @awaelchli @ethanwharris
```

**Comment by Borda** [comment]
> I ran the quick start app manually when submitting this PR a few weeks ago: [Lightning-Universe/lightning-quick-start#30](https://github.com/Lightning-Universe/lightning-quick-start/pull/30) and it ran fine. Do you think the issue is here in our testing?

That repo does not have set any related secrets/credentials so I don't this the PR there was running on the production backend

**Comment by Borda** [comment]
> I am not sure whether we should really skip this app. It seems to be working fine when run manually as a regular app.

Then we shall reconsider/re-evaluate how we are testing these apps...
this app in particular was flaky for months and is constantly failing for last few weeks

**Review by lantiga — APPROVED** [review]

**Pull request merged**
